rmf_gazebo_plugins/src/teleport.cpp

```diff
@@ -93,6 +96,13 @@ class TeleportPlugin : public gazebo::ModelPlugin
       RCLCPP_INFO(_node->get_logger(),  "Unloading object");
       _model->SetWorldPose(_unload_pose);
       _object_loaded = false;
+
+      // Hard coded: Leave object at goal location for 2.0 second, then
+      // teleport
```

Comment:
Is there a reason for breaking the comment after "teleport"? If not can we fit the comments into two lines?